configure-clusters/src/main/java/com/esri/arcgismaps/sample/configureclusters/screens/MainScreen.kt

```diff
@@ -0,0 +1,398 @@
+                            .align(Alignment.BottomEnd)
+                            .padding(bottom = 36.dp, end = 24.dp),
+                        onClick = {
+                            composableScope.launch {
+                                controlsBottomSheetState.show()
+                            }
+                        },
+                    ) {
+                        Icon(Icons.Filled.Settings, contentDescription = "Show controls")
+                    }
+                }
+                if (controlsBottomSheetState.isVisible) {
+                    ClusterControlsBottomSheet(
+                        composableScope = composableScope,
+                        controlsBottomSheetState = controlsBottomSheetState,
+                        showClusterLabels = mapViewModel.showClusterLabels,
+                        updateClusterLabelState = mapViewModel::updateShowClusterLabelState,
+                        clusterRadiusOptions = mapViewModel.clusterRadiusOptions,
+                        clusterRadius = mapViewModel.clusterRadius,
+                        updateClusterRadiusState = mapViewModel::updateClusterRadiusState,
+                        clusterMaxScaleOptions = mapViewModel.clusterMaxScaleOptions,
+                        clusterMaxScale = mapViewModel.clusterMaxScale,
+                        updateClusterMaxScaleState = mapViewModel::updateClusterMaxScaleState,
+                        mapScale = mapScale
+                    )
+                }
+            }
+
+            // display a bottom sheet to show popup details
+            BottomSheet(
+                isVisible = mapViewModel.showPopUpContent,
+                bottomSheetContent = {
+                    ClusterInfoContent(
+                        popUpTitle = mapViewModel.popUpTitle,
+                        popUpInfo = mapViewModel.popUpInfo,
+                        onDismiss = { mapViewModel.updateShowPopUpContentState(false) }
+                    )
+                })
+
+        }
+    )
+}
+
+/**
+ * Composable function to display the cluster controls bottom sheet.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun ClusterControlsBottomSheet(
+    composableScope: CoroutineScope,
+    controlsBottomSheetState: SheetState,
+    showClusterLabels: Boolean,
+    updateClusterLabelState: (Boolean) -> Unit,
+    clusterRadiusOptions: List<Int>,
+    clusterRadius: Int,
+    updateClusterRadiusState: (Int) -> Unit,
+    clusterMaxScaleOptions: List<Int>,
+    clusterMaxScale: Int,
+    updateClusterMaxScaleState: (Int) -> Unit,
+    mapScale: Int,
+) {
+    ModalBottomSheet(
+        modifier = Modifier.wrapContentHeight(),
+        sheetState = controlsBottomSheetState,
+        onDismissRequest = {
+            composableScope.launch {
+                controlsBottomSheetState.hide()
+            }
+        }) {
+        Row {
+            Column(
+                Modifier
+                    .padding(12.dp)
+            ) {
```

Comment:
I noticed an issue with the `ModalBottomSheet` displaying behind the navigation bar, causing some of the content to be cut-off. I tested this on the latest Compose BOM, and it seems to still persist. There is an issueTracker [link](https://issuetracker.google.com/issues/305565075), but doesn't seem like this will be fixed anytime soon. 

![Screenshot 2024-06-13 at 4 57 03 PM](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/88b3c203-c876-45f5-8a08-9fd3ee617a7e)

For a fix, we can use `.navigationBarsPadding()` modifier on the parent layout to ensure the entire sheet is always visible. 

```suggestion
            Column(Modifier.padding(12.dp).navigationBarsPadding()) {
```

- With Modifier applied: 

![Screenshot 2024-06-13 at 4 56 07 PM](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/e2c474c9-fd02-46ce-aff9-bd4032ba440c)

Reply:
Haha I spent 30 min yesterday evening trying to bump it up, but I was using the Cluster Max Scale's padding to try to accomplish it (unsuccessfully!). This did the trick, nice one!